spec/html_preprocessor_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+describe PDFKit::HTMLPreprocessor do
```

Comment:
Dan added that test and a few other test cases. I consider this to be addressed.